plugins/Kaleidoscope-DynamicMacros/src/kaleidoscope/plugin/DynamicMacros.h

```diff
@@ -50,7 +50,8 @@ class DynamicMacros : public kaleidoscope::Plugin {
   static uint16_t storage_base_;
   static uint16_t storage_size_;
   static uint16_t map_[31];
```

Comment:
It's not unrelated, because this PR is supposed to fix bugs in DynamicMacros, and this is a quick and easy fix for a clear bug, so I'll just push an update.

Thanks!